src/scandipwa/app/plugin/event-dispatches/cartItemChangeQuantity.plugin.js

```diff
@@ -19,32 +19,38 @@ class QuantityChangePlugin {
         };
 
         callback(...args);
-    }
+    };
 
     changeItemQty = (args, callback, instance) => {
+        if (!isEventEnabled(EVENT_ADD_TO_CART)) return callback(...args);
+
         const { newQuantity, item, oldQuantity } = this.handleChangeState;
 
         return callback(...args)
             .then(
                 result => {
                     if (oldQuantity < newQuantity) {
-                        Event.dispatch(EVENT_GTM_PRODUCT_ADD_TO_CART, {
-                            product: item,
-                            quantity: newQuantity - oldQuantity,
-                            isItem: true,
-                            isFromCart: true
-                        });
+                        if (isEventEnabled(EVENT_ADD_TO_CART)) {
+                            Event.dispatch(EVENT_GTM_PRODUCT_ADD_TO_CART, {
+                                product: item,
+                                quantity: newQuantity - oldQuantity,
+                                isItem: true,
+                                isFromCart: true
+                            });
+                        }
```

Comment:
I propose to move the `isEventEnabled` checks to the `Event.dispatch` function itself, so that you need to do that check only once. This way the code would be easier to be maintained.